flow-model-generator/modules/flow-model-generator-core/src/main/java/io/ballerina/flowmodelgenerator/core/DeleteNodeHandler.java

```diff
@@ -105,6 +113,32 @@ private static JsonElement getTextEditsToDeletedNode(LineRange lineRange, Path f
                 ImportDeclarationNode importNode = getUnusedImport(diagnostic.location().lineRange(), imports);
                 TextEdit deleteImportTextEdit = new TextEdit(CommonUtils.toRange(importNode.lineRange()), "");
                 textEdits.add(deleteImportTextEdit);
+
+                List<DiagnosticProperty<?>> diagnosticProperties = diagnostic.properties();
+                if (diagnosticProperties != null && !diagnosticProperties.isEmpty()) {
+                    String diagnosticProperty = diagnosticProperties.getFirst().value().toString();
+                    if (dbDrivers.contains(diagnosticProperty)) {
+                        String expectedModuleName = diagnosticProperty + ".driver";
+                        String expectedPrefix = "as _";
+                        for (ImportDeclarationNode importDeclarationNode : imports) {
+                            var orgName = importDeclarationNode.orgName();
+                            var prefix = importDeclarationNode.prefix();
```

Comment:
There are only limited use cases where using the `var` keyword is justifiable (such as avoiding type repetition in the right-hand side or saving an intermediate stream step). In this case, it's better to explicitly write the type (you can use the code action for this), as using `var` compromises code readability since the actual type of `orgName()` is not immediately clear.